docs/_docs/tutorial/01-your-first-lte.md

```diff
@@ -226,58 +220,67 @@ Modify [install/etc/open5gs/mme.yaml](https://github.com/{{ site.github_username
 
 ```diff
 $ diff -u /etc/open5gs/mme.yaml.old /etc/open5gs/mme.yaml
---- mme.yaml.old	2020-08-22 12:07:32.755250028 -0400
-+++ mme.yaml	2020-08-22 12:08:17.309320211 -0400
-@@ -208,20 +208,20 @@
- mme:
-     freeDiameter: /home/acetcom/Documents/git/open5gs/install/etc/freeDiameter/mme.conf
-     s1ap:
--      addr: 127.0.0.2
-+      addr: 127.0.1.2
-     gtpc:
-       addr: 127.0.0.2
+--- mme.yaml.old 2022-01-19 10:36:02.462763600 +0100
++++ mme.yaml    2022-01-19 10:37:21.862156800 +0100
+@@ -213,14 +213,14 @@
+       - addr: 127.0.0.2
      gummei:
        plmn_id:
 -        mcc: 901
 -        mnc: 70
-+        mcc: 310
-+        mnc: 789
++        mcc: 901
++        mnc: 70
```

Comment:
the change makes this diff look kinda weird, but my intention in keeping it here and highlighted by diff is, that people using another MCC/MNC will know where all they will have to change these values.